topology/graph/message.go

```diff
@@ -87,7 +87,7 @@ func UnmarshalWSMessage(msg *shttp.WSJSONMessage) (string, interface{}, error) {
 			syncRequest.TimeSlice = common.NewTimeSlice(i, i)
 		}
 
-		if s, ok := m["GremlinFilter"]; ok {
+		if s, ok := m["topologyFilter"]; ok {
```

Comment:
We should better stick with "GremlinFilter" as it's the attribute that will be used when marshalling a SyncMsgRequest to JSON.

You will need to change the line 692 of statics/js/components/topology.js too

Reply:
OK, I am doing it

Reply:
Done